test/e2e/user_workloads_test.go

```diff
@@ -0,0 +1,116 @@
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	"github.com/codeready-toolchain/api/pkg/apis/toolchain/v1alpha1"
+	. "github.com/codeready-toolchain/toolchain-e2e/testsupport"
+	"github.com/codeready-toolchain/toolchain-e2e/wait"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func TestUserWorkloads(t *testing.T) {
+	suite.Run(t, &userWorkloadsTestSuite{})
+}
+
+type userWorkloadsTestSuite struct {
+	baseUserIntegrationTest
+}
+
+func (s *userWorkloadsTestSuite) SetupSuite() {
+	s.ctx, s.hostAwait, s.memberAwait = WaitForDeployments(s.T(), &v1alpha1.UserSignupList{})
+	s.setApprovalPolicyConfig("automatic")
+}
+
+func (s *userWorkloadsTestSuite) TearDownTest() {
+	s.ctx.Cleanup()
+}
+
+func (s *userWorkloadsTestSuite) TestIdler() {
```

Comment:
```suggestion
func (s *userWorkloadsTestSuite) TestIdlerOnDeployment() {
```

Reply:
I was planning to use this function for other payloads too.